NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Em database

## Description of Proposed Changes

Adds SQL script to delete and rebuild the database structure

## Steps to Test

```sh
git fetch --all
git checkout EM_Database
```
1. Open bangazon database in DB Browser if it exists
1. Copy script in ./bangazon_database/reset_database.sql
1. Paste SQL script into DB Browser & run it

Expected Result:

The tables in the database should be deleted and recreated without errors.
